crates/rolldown/src/stages/scan_stage.rs

```diff
@@ -140,17 +139,15 @@ impl ScanStage {
           ResolveError::PackagePathNotExported(..) => {
             errors.push(BuildDiagnostic::unresolved_entry(args.specifier, Some(e)));
           }
-          _ => {
-            return Err(e.into());
-          }
+          _ => return Err(e)?,
         },
       }
     }
 
     if !errors.is_empty() {
-      return Ok(Err(errors.into()));
+      Err(errors)?;
```

Comment:
Here look like missing `return`.

Reply:
Actually, this is syntactic sugar. `return Err(errors.into());` and `Err(errors)?;` are equivalent, because the `?` operator automatically performs both the `return` and `into` conversion for an `Error`.

Reply:
This might be confused, I will add `return` to them. Since it was also used in the previous PRs, I will open a separate PR to resolve it.

Reply:
I get it, i'm not notice it. I think it's ok.